flux-build-scripts/update-flux-versions.js

```diff
@@ -89,7 +103,7 @@ var fluxVersions = {};
   await getAllFluxVersions(enterpriseVersionArr, 'enterprise', enterpriseVersions);
 
   // Manually add versions that aren't included in the original versions arrays
-  await addVersion(ossVersions, 'nightly', await getFluxVersion('master'));
+  await addVersion(ossVersions, 'nightly', await getFluxVersion('2.7'));
   await addVersion(enterpriseVersions, '1.8', await getFluxVersion('1.8'));
   await addVersion(enterpriseVersions, '1.7', await getFluxVersion('1.7'));
```

Comment:
More magic numbers?

Reply:
I didn't know what to do here. The `master` branch is no longer maintained (because the switched the primary branch to `main`) and the v2 nightlies are being build from the 2.7 branch.